apph-back/src/main/java/com/viseo/apph/service/PhotoService.java

```diff
@@ -177,4 +177,18 @@ public MessageListResponse movePhotosToFolder(User user, PhotosRequest request)
         response.addMessage("success: Le déplacement des photos est terminé.");
         return response;
     }
+
+    @Transactional
+    public String changePhotoFile(Long userId, PhotoRequest photoRequest) throws IOException, UnauthorizedException, InvalidFileException {
+        Photo photo = photoDao.getPhoto(photoRequest.getId());
+        if (photo == null) {
+            throw new FileNotFoundException("Le fichier n'existe pas");
+        }
+        if (userId != photo.getUser().getId()) {
+            throw new UnauthorizedException("L'utilisateur n'est pas autorisé à accéder à la ressource demandée");
+        }
+        s3Dao.delete(photo);
+        photo.setFormat(getFormat(photoRequest.getFile())).setSize((photoRequest.getFile().getSize() + .0F) / 1000);
```

Comment:
```suggestion
        photo.setFormat(getFormat(photoRequest.getFile())).setSize((photoRequest.getFile().getSize() + .0F) / 1024);
```
Je m'étais trompé sur le calcul